model/testresult/testresult.go

```diff
@@ -100,6 +80,15 @@ func (t *TestResult) Insert() error {
 	return db.Insert(Collection, t)
 }
 
+func InsertMany(results []TestResult) error {
+	docs := make([]interface{}, len(results))
```

Comment:
Is this needed? Why not pass results directly to db.InsertMany?

Reply:
I wish that worked: https://play.golang.org/p/YTJXj_60HG1